torch/utils/_content_store.py

```diff
@@ -120,7 +120,6 @@ def kernel(x):
         if pad > 0:
             x = F.pad(x, (0, pad), "constant", 0)
         x = x.view(torch.int32)
-        torch._dynamo.mark_dynamic(x, 0)
```

Comment:
Why do we not want this anymore? (is `mark_dynamic` not robust to dims known to be size 0/1?)

Reply:
I need to fix the constraint checker but it is a more involved change

Reply:
Right now it errors saying we couldn't make the 1 size dim dynamic.